### PR TITLE
feat(io): edits to Snowflake IO page and created blazor web assembly page

### PR DESCRIPTION
**Pull request by akristen**

### CODE REVIEW

**Comment by paperclypse** [comment]
@akristen Looks like you might need to add the Blazor doc to the nav?

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/blazor-web-assembly-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+title: Snowflake integration
```
Comment (paperclypse) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/blazor-web-assembly-integration.mdx:2]: Shouldn't this say Blazor web assembly or something?

**Review by paperclypse — DISMISSED** [review]
Left some minor comments on codestream @akristen

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx [hunk 2]
```diff
@@ -931,12 +931,27 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
           <td>
 If your application uses ADO.NET, you can install a combination of New Relic agents that will generate an ADO dashboard. Here’s what you need to do:
```
Comment (pkosanam) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:932]: If your application uses ADO.NET, you can install a combination of New Relic agents that will generate an ADO .net dashboard. Here’s what you need to do:

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx [hunk 3]
```diff
@@ -931,12 +931,27 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
...
         </tr>
+        <tr>
+          <td>
+            Blazor WebAssembly
+          </td>
+          <td>
+            To monitor Blazor WebAssembly, you need to install the .NET agent, then install the .NET quickstart from our instant observability page. Here's what you need to do:
+
+            1. Install the New Relic APM .NET agent and the infrastructure agent by following the steps in our [guided installation](https://one.newrelic.com/marketplace?state=bf68ca4b-b7cc-3227-fcc3-3349f5aa92f6). Be sure to restart your application to complete the installation.
+            2. Confirm that your data is in New Relic by going to [one.newrelic.com](https://one.newrelic.com/all-capabilities), then clicking **Metrics & events**. If your data is in New Relic, you should be able to query your Blazor WebAssembly data.
```
Comment (pkosanam) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:948]: 2nd point can be replaced with the below text

Navigate to the “APM & service monitoring” section through the navigation bar and then look for your application name. On choosing the application, you should see all the metrics which are captured
You can choose our pre-built dashboard template of “Blazor Server” to monitor your Blazor Server application metrics.

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx [hunk 4]
```diff
@@ -931,12 +931,27 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
...
+            3. Go to our [.NET instant observability page](https://newrelic.com/instant-observability/dotnet), then click **Install now**.
```
Comment (pkosanam) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:949]: 3rd point can give the example query

Example query:
FROM Transaction SELECT count(*) FACET request.uri

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx [hunk 5]
```diff
@@ -931,12 +931,27 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
...
+          </td>
+          <td>
+            To monitor Blazor WebAssembly, you need to install the .NET agent, then install the .NET quickstart from our instant observability page. Here's what you need to do:
+
+            1. Install the New Relic APM .NET agent and the infrastructure agent by following the steps in our [guided installation](https://one.newrelic.com/marketplace?state=bf68ca4b-b7cc-3227-fcc3-3349f5aa92f6). Be sure to restart your application to complete the installation.
+            2. Confirm that your data is in New Relic by going to [one.newrelic.com](https://one.newrelic.com/all-capabilities), then clicking **Metrics & events**. If your data is in New Relic, you should be able to query your Blazor WebAssembly data.
+            3. Go to our [.NET instant observability page](https://newrelic.com/instant-observability/dotnet), then click **Install now**. 
+            4. Choose your Blazor WebAssembly account as the account you want to monitor. 
+
+            Once installed, you'll find your dashboards by going to [one.newrelic.com](https://one.newrelic.com/all-capabilities), then clicking the **Dashboards** tile.
```
Comment (pkosanam) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:952]: ## Get Blazor Server metrics in a dashboard
Once you've installed the [Blazor Server quickstart](https://newrelic.com/instant-observability/blazor-server), you can see your critical Blazor Server data in New Relic.

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx [hunk 6]
```diff
@@ -931,12 +931,27 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
...
+          <td>
+            Blazor WebAssembly
+          </td>
+          <td>
+            To monitor Blazor WebAssembly, you need to install the .NET agent, then install the .NET quickstart from our instant observability page. Here's what you need to do:
+
+            1. Install the New Relic APM .NET agent and the infrastructure agent by following the steps in our [guided installation](https://one.newrelic.com/marketplace?state=bf68ca4b-b7cc-3227-fcc3-3349f5aa92f6). Be sure to restart your application to complete the installation.
+            2. Confirm that your data is in New Relic by going to [one.newrelic.com](https://one.newrelic.com/all-capabilities), then clicking **Metrics & events**. If your data is in New Relic, you should be able to query your Blazor WebAssembly data.
+            3. Go to our [.NET instant observability page](https://newrelic.com/instant-observability/dotnet), then click **Install now**. 
+            4. Choose your Blazor WebAssembly account as the account you want to monitor.
```
Comment (pkosanam) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:950]: As per my understanding, this line can be removed

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx (hunk 5 above)
Comment (akristen) [reply src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:952]: Are they installing the quickstart from the .NET IO page, or from this independent blazor page?

Comment (akristen) [reply src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:952]: Ah, wait, this is a separate quickstart I think

Comment (pkosanam) [reply src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx]: my bad. I am sorry for the confusion.

Blazor Server is the one which uses the .net agent
Blazor WebAssembly is the one which uses the browser monitoring agent

**Comment by pkosanam** [comment]
In the PR name, "Blazor web assembly" has to be replaced with "Blazor Server"

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx [hunk 7]
```diff
@@ -931,12 +931,34 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
...
 3. If you installed Perfmon manually, [restart your infrastructure agent](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent).
 4. Go to our ADO.NET quickstart page to [install the ADO.NET dashboards](https://newrelic.com/instant-observability/ado-dotnet).
           </td>
         </tr>
+        <tr>
+          <td>
+            Blazor WebAssembly
+          </td>
+          <td>
+            To monitor Blazor WebAssembly, you need to install the .NET agent, then install the .NET quickstart from our instant observability page. Here's what you need to do:
```
Comment (pkosanam) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:945]: To monitor Blazor Server, you need to install the .NET agent, then install the .NET quickstart from our instant observability page. Here's what you need to do:

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx [hunk 8]
```diff
@@ -931,12 +931,34 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
...
+
+            1. Install the New Relic APM .NET agent and the infrastructure agent by following the steps in our [guided installation](https://one.newrelic.com/marketplace?state=bf68ca4b-b7cc-3227-fcc3-3349f5aa92f6). Be sure to restart your application to complete the installation.
+            2. To confirm that your data's in New Relic, go to [one.newrelic.com](https://one.newrelic.com/all-capabilities), click **Metrics & events**, then select your application. To check your captured metrics, try this query: `FROM Transaction SELECT count(*) FACET request.uri`.
+            3. To get our quickstart dashboards, go to our [.NET instant observability page](https://newrelic.com/instant-observability/dotnet), then click **Install now**.
```
Comment (pkosanam) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:949]: Once .net agent is successfully integrated, your Blazor Server metrics are sent to one.newrelic.com. To install your dashboard.
1. go to Add Data from one.newrelic.com
2. Search for Blazor Server in Dashboards section
3. Install the Blazor Server dashboard
4. Then you can see your dashboard in [one.newrelic.com](https://one.newrelic.com/all-capabilities) -> ***Dashboards*** section

**Review by pkosanam — CHANGES_REQUESTED** [review]
the content in this PR is related to Blazor Server and the PR name has to be changed

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx (hunk 5 above)
Comment (akristen) [reply src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx]: AH! Ok, that was my bad too. I will make my changes :)

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx [hunk 9]
```diff
@@ -931,14 +931,32 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
...
         </tr>
+        <tr>
+          <td>
+            Blazor Server
+            </td>
+          <td>
+Track your most important data in our Blazor Server dashboard by instrumenting your app with New Relic: 
+
+1. Install the New Relic APM .NET agent and the infrastructure agent by following the steps in our [guided installation](https://one.newrelic.com/marketplace?state=bf68ca4b-b7cc-3227-fcc3-3349f5aa92f6).If you prefer, you can [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) via the command line.
+2. Get the pre-built dashboard by [installing the Blazor quickstart](https://newrelic.com/instant-observability/blazor-server)
```
Comment (pkosanam) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx:950]: Can rename "Blazor quickstart" to "Blazor Server quickstart"

**Review by pkosanam — CHANGES_REQUESTED** [review]

**Comment by pkosanam** [comment]
The PR name should be "feat(io): edits to Snowflake IO page and created blazor server page"

**Review by paperclypse — DISMISSED** [review]
I'm approving this because I'm just going to assume you'll address all my codestream feedback :)

**Review by homelessbirds — APPROVED** [review]

**Review by pkosanam — APPROVED** [review]
looks good

**Pull request merged**